Trappist/src/Promact.Trappist.Repository/TestSettings/TestSettingsRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Models.Test;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Promact.Trappist.Repository.TestSettings
+{
+    public class TestSettingsRepository : ITestSettingsRepository
+    {
+        private readonly TrappistDbContext _dbContext;
+        Test test = new Test();
+
+        public TestSettingsRepository(TrappistDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        /// <summary>
+        /// Updates the changes made to the Settings of a Test
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="testObject"></param>
+        /// <returns>Updated Setting of that Test</returns>
+        public string UpdateTestSettings(int id, Test testObject)
+        {
+            testObject = new Test();
+            test = (from s in _dbContext.Test where s.Id == id select s).FirstOrDefault();
```

Comment:
Default value will that is null in this case will be displayed in the respective fields.